libc/hdr/types/CMakeLists.txt

```diff
@@ -126,3 +126,12 @@ add_proxy_header_library(
     libc.include.llvm-libc-types.atexithandler_t
     libc.include.stdlib
 )
+
+add_proxy_header_library(
+  struct_sigaction
+  HDRS
+    struct_sigaction.h
+  FULL_BUILD_DEPENDS
+    libc.include.llvm-libc-types.struct_sigaction
+    libc.include.signal
```

Comment:
Why do we need `libc.include.signal`?

Reply:
I think it is - convention in proxy headers to require the full build to generate the outer-level header. This is employed in all other proxy type headers. @lntue

Reply:
Approving for now, but perhaps worth revisiting this convention. @lntue ?

Reply:
merge this for now. if the convention is revised, we should better revisit all other files.